integrations/spark/openhouse-spark-runtime/build.gradle

```diff
@@ -61,7 +60,7 @@ dependencies {
   fatJarPackagedDependencies(project(path: ':integrations:java:openhouse-java-runtime', configuration: 'shadow')) {
     transitive = false
   }
-  fatJarRuntimeDependencies("org.apache.iceberg:iceberg-spark-runtime-3.1_2.12:" + icebergVersion)
+  implementation("org.apache.iceberg:iceberg-spark-runtime-3.1_2.12:" + icebergVersion)
 
   generateGrammarSource {
     maxHeapSize = "64m"
```

Comment:
![image](https://github.com/user-attachments/assets/ee788a85-71ee-41d2-9a49-6aae3c41d2c2)

was this library getting pulled in before as well? if not, we'd need to be careful with deployments

Reply:
Yes this lib was pulled in before as well as runtime dependency.